src/simulation/m_global_parameters.fpp

```diff
@@ -891,21 +971,33 @@ contains
         intxe = internalEnergies_idx%end
 
         !$acc update device(momxb, momxe, advxb, advxe, contxb, contxe, bubxb, bubxe, intxb, intxe, sys_size, buff_size, E_idx, alf_idx, strxb, strxe)
+        !$acc update device(m, n, p)
```

Comment:
is this right for nvhpc?

Reply:
Hmm I don't see how this could be an issue, there might be duplicate updates but that shouldn't make a difference anyways